client/app/views/templates/popover_screens/guest_row.jade

```diff
@@ -12,14 +12,45 @@ li.guest-top(data-index=index)
     .guest-label= label
 
     if !readOnly
-        span.button-wrapper
-            button.guest-delete.fa.fa-trash-o(title=t('screen guest remove tooltip'), role="button")
+        button.guest-delete.fa.fa-trash-o(
+            title=t('screen guest remove tooltip')
+            role="button"
+        )
 
-        // If a cozy instance is linked to a contact
+        - var hideShare    = true
+        - var hideEmail    = true
+        - var activeShare  = false
+        - var activeEmail  = false
+
+        // If a cozy instance is linked to a contact.
         if cozy
+            - hideShare = false
+
+            // The invitation is shared.
             if shareWithCozy
-                span.button-wrapper
-                    button.guest-share-with-email.fa.fa-envelope-o(title=t('screen guest share with email tooltip'), role="button")
-            else
-                span.button-wrapper
-                    button.guest-share-with-cozy.fa.fa-cloud(title=t('screen guest share with cozy tooltip'), role="button")
+                - activeShare  = true
+
+        // If the contact also has an email.
+        if email
+            - hideEmail = false
+
+            // The invitation is not shared, hence sent by email.
+            if !shareWithCozy
+                - activeEmail  = true
+
```

Comment:
I feel that there is too much logic and variable affectation here, as we are in a template.

What about passing all this conditional stuff in the method `renderAttendees` in `GuestPopoverScreen` ?